### ISSUE TITLE
ReceivablesList returns amount in default currency

### ISSUE THREAD

**Issue by somegooser** [picqer/exact-php-client#650]
Hi,

I use the ReceivablesList endpoint to retrieve all outstanding payments. The problem I am running into is that the amounts I receive back are in EUR while the payment/invoice itself is booked in a different currency (USD). How can I receive the outstanding amounts in the currency of the booking itself?

**Comment by remkobrenters** [comment]
I know that often Exact has an amount in the default currency (Amount DC) and in the selected currency  (Amount FC) but this endpoint https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=ReadFinancialReceivablesList works differently. Is the currencyCode EUR in this case or USD?

**Comment by somegooser** [comment]
Yes the CurrencyCode is EUR. When you fetch the SalesEntry endpoint with the given InvoiceNumber it actually says the invoice is in USD. So i cant 'read' the OutstandingAmount because the currency is in EUR while the Invoice is in USD.

**Comment by remkobrenters** [comment]
And if you lookup the same payment through https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=BulkCashflowReceivables does it include the expected value in the AmountFC field for example?

**Comment by somegooser** [comment]
Thank you. The amount here is in the currency of the invoice. So thats a good thing! 

Is the value in the 'Amount' the outstanding amount? I cant find any info about that in the API docs.

**Comment by remkobrenters** [comment]
Good. I cannot tell you if that is the outstanding amount or the total amount from the top of my mind. Not sure if anyone else knows this?

**Comment by remkobrenters** [comment]
@somegooser did you found out and is  there something you can share about it to resolve this issue but also as a reference for people in the future who run into similar questions?
